code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1044,46 +1043,41 @@
 	reac_volume = round(reac_volume,0.1)
 	exposed_turf.acid_act(acidpwr, reac_volume)
 
+// when you ingest acid (not the fun kind) death awaits... 
+/datum/reagent/toxin/acid/on_mob_life(mob/living/carbon/victim, delta_time, times_fired)
+	var/acid_dmg = acidpwr * 0.1 // results in 1 for sulphric acid, 4.2 for flurosuluric acid, and 0.5 for nitric acid 
+	victim.adjustBruteLoss(acid_dmg * REM * delta_time, 0)
+	victim.adjustFireLoss(acid_dmg * 2 * REM * delta_time, 0) // take double burn damage
+
+	if(DT_PROB(16, delta_time))
+		victim.adjustOrganLoss(ORGAN_SLOT_STOMACH, acid_dmg * REM * delta_time)
+		victim.emote("gurgle")
+	else if(DT_PROB(16, delta_time))
+		victim.adjustOrganLoss(ORGAN_SLOT_LIVER, acid_dmg * REM * delta_time)
+		victim.emote("grimace")	
+	..()
+	//return TRUE  (is this needed?)
```

Comment:
IIRC the return TRUE bit prompts the mob to update its health at the end of metabolization. It's done that way to prevent update spam I believe.